docs/02-guides/display-prices-with-or-without-taxes.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: "Display prices with or without taxes"
+hide_title: true
+description:
+  This guide walks through the process of displaying prices with or without
+  taxes within your Front-Commerce application.
+---
+
+## Display prices with or without taxes
+
+<SinceVersion tag="3.9" />
+
+To display a price without taxes, you can add a new
+[public configuration](/docs/3.x/guides/configuration/public-configuration) in
+your application by extending the config providers.
+
+See
+[example](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/a06c7bcf9a1e759a366abee1b1e34eb7daf081fa/skeleton/example-extensions/price-variant-demo/index.ts)
+below:
+
+```ts
+import { defineExtension } from "@front-commerce/core";
+import type { PublicConfig } from "@front-commerce/core/react";
+
+type PriceDisplayConfig = PublicConfig["theme"]["priceDisplay"];
+
+const priceDisplayConfigProvider = {
+  name: "price-variant-demo:public",
+  values: {
+    public: {
+      theme: {
+        priceDisplay: {
+          variants: ["priceInclTax", "priceExclTax"],
+          appearance: "labeled",
+        } satisfies PriceDisplayConfig,
+      },
+    },
+  },
+};
+
+export default defineExtension({
+  name: "price-variant-config",
+  meta: import.meta,
+  configuration: {
+    providers: [priceDisplayConfigProvider],
+  },
+});
+```
+
+:::info
+
+The available options for the `priceDisplay` configuration are:
+
+- `variants`: the list of price variants to display, available variants are
+  `priceInclTax` and `priceExclTax`.
+- `appearance`: the appearance of the price, available appearances are `labeled`
+  and `default`.
+  - `labeled`: this includes the label `incl. tax` or `excl. tax`
+  - `default`: this does not include any label
+
+:::
+
+## Preview
```

Comment:
**thought:** I wonder if this shouldn't be relocated to a `Theme Chocolatine > Reference >  Prices display` section

![2024-11-04_10-04](https://github.com/user-attachments/assets/084bea6b-8b1a-4960-9be7-39b4bd6c1521)

I let you decide.